src/modules/Lan.rb

```diff
@@ -894,13 +839,54 @@ def activate_network_service
     # It clears already read configuration.
     def read_config
       system_config = Y2Network::Config.from(:sysconfig)
+      system_config.backend = NetworkService.cached_name
```

Comment:
i think this should be hidden in the factory itself

Reply:
It is not sysconfig especific, so I preferred to left it out of the reader. Thus, would you prefer to move it?

Reply:
well it simply raises a question in my head whether we can simply rely on having this separated readers or whether we need even some shared part when creating config storage